pkgs/top-level/python-packages.nix

```diff
@@ -16810,19 +16810,24 @@ EOF
   };
 
   tflearn = buildPythonPackage rec {
-    name = "tflearn-0.2.1";
+    version = "0.3.2";
+    name = "tflearn-${version}";
 
-    meta = {
-      description = "Deep learning library featuring a higher-level API for TensorFlow";
-      homepage    = "https://github.com/tflearn/tflearn";
-      license     = licenses.mit;
+    src = pkgs.fetchurl {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
Example refactoring: https://github.com/NixOS/nixpkgs/pull/45763